### DISCUSSION TITLE
Make an Automated Task To Convert Email In Plain HTML From React Component

### DISCUSSION THREAD

**Discussion by nishant23122000** [resend/react-email#692] [Q&A]
I have created one repository for email templates developed in react-email,
and I have another backend repository where I have to split the whole HTML email template in a different layout, 
So, is there any way anyone can change the react-email react code and it automatically raises PR or informs in backend repository about the change made in the react code?

**Comment by dyaa** [comment]
>  I have to split the whole HTML email template in a different layout

Would you please elaborate on that?

---
If I understood correctly, your inquiry is not within the scope of react-email itself. It would be around your CI/CD setup, So this is how I imagine achieving that.

1. You update an email template and push the changes _somehow_ to the main branch
1. CI build your email template and CD to publish these artifacts _somewhere_.
1. Backend repository to consume these HTML artifacts and keep them up-to-date; for example, pull artifacts every X period of time, CD part from the previous point to call a webhook that triggers a job of pulling/refreshing the artifacts.

Let me know what you think or if I was able to help :)
